aiida-registry-app/src/App.jsx

```diff
@@ -53,6 +58,31 @@ function App() {
 }
 
 function MainIndex() {
+  const [sortOption, setSortOption] = useState('alpha');
+  const [sortedData, setSortedData] = useState(plugins);
+  const handleSort = (option) => {
+    setSortOption(option);
+
+
+    let sortedPlugins;
+    if (option === 'commits') {
+      const pluginsArray = Object.entries(plugins);
+
+      // Sort the array based on the commit_count value
+      pluginsArray.sort(([, pluginA], [, pluginB]) => pluginB.commits_count - pluginA.commits_count);
+
+      // Create a new object with the sorted entries
+      sortedPlugins = Object.fromEntries(pluginsArray);
```

Comment:
Very nice, it is pretty clear, I am not very familiar with js and I can understand it in the first glimpse.